src/main/java/net/blay09/mods/waystones/Waystones.java

```diff
@@ -38,12 +39,16 @@
 	public static ItemWarpScroll itemWarpScroll;
 	public static ItemWarpStone itemWarpStone;
 
+	public static WaystoneCreativeTab creativeTab;
+
 	public static Configuration configuration;
 
 	private WaystoneConfig config;
 
 	@Mod.EventHandler
 	public void preInit(FMLPreInitializationEvent event) {
+		creativeTab = new WaystoneCreativeTab();
```

Comment:
Should be assigned at declaration to keep it more contained, no need to create it in preInit